scripts/release-hooks.mjs

```
@@ -8,6 +8,14 @@ if (!version) {
   process.exit(1);
 }
 
+// release-it runs before:bump hooks even in dry-run mode, which would otherwise
+// modify real package.json / manifest.json files and stage them with `git add`.
+// `scripts/release.mjs` sets RELEASE_IT_DRY_RUN=1 when --dry-run is passed.
+if (process.env.RELEASE_IT_DRY_RUN === "1") {
```

ESLint

[error] 'process' is not defined.

(no-undef)